### PR TITLE
Development

### PR DESCRIPTION
**Pull request by TusharW4ni**
New frontend, backend, database schema, and database migrations.

### CODE REVIEW

File: backend/clientConnection.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Will use .env in the future. For now using this.
+
+const { Client } =  require("pg")
```
Comment (MStolen) [inline backend/clientConnection.js]: This looks like you're using [node-postgres](https://www.npmjs.com/package/pg), but you also have [Prisma](https://prisma.io) migrations and a schema. I'm a little confused why you're using both.

File: backend/prisma/migrations/20230930231127_initial/migration.sql
Comment (MStolen) [inline backend/prisma/migrations/20230930231127_initial/migration.sql:1]: I'm not sure if you need to include any of your Prisma migrations in your repository. @pariahGH, please comment.

File: backend/clientConnection.js (hunk 1 above)
Comment (MStolen) [reply backend/clientConnection.js]: Looking further, it seems like this isn't really being used and you are using Prisma. Can this file be deleted?

**Review by MStolen — DISMISSED** [review]
@TusharW4ni, I approve but have a couple comments. Taz can weigh in more about the Prisma migrations. It looks like there's legacy stuff from node-postgres, but it doesn't look like you actually use it.

File: backend/clientConnection.js (hunk 1 above)
Comment (TusharW4ni) [reply backend/clientConnection.js]: You are correct, the clientConnection.js file can be removed. I didn't pay attention to it before.

**Review by MStolen — APPROVED** [review]

**Pull request merged**